manifests/overlays/kubeflow/kustomization.yaml

```diff
@@ -7,3 +7,9 @@ resources:
 images:
   - name: kubeflow/training-operator
     newTag: "v1-855e096"
+# TODO (tenzen-y): Once we support cert-manager, we need to remove this secret generation.
+# REF: https://github.com/kubeflow/training-operator/issues/2049
+secretGenerator:
+  - name: training-operator-webhook-secret
```

Comment:
For the secret name, would it be better to just name it: `training-operator-webhook`. Similar to [Katib](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-standalone/kustomization.yaml#L39) and not add the Kubernetes resource name (e.g. `secret`) to the name ?

Reply:
Oh, you already mentioned here: https://github.com/kubeflow/training-operator/pull/2035#discussion_r1553933932

> Secret name: training-operator-webhook-cert

I'll try to refine this name.

Reply:
Done.